kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -385,12 +389,20 @@ def intlist(ints: list[KInner]) -> KApply:
         return res
 
     @staticmethod
-    def typed_args(args: list[KInner]) -> KApply:
+    def typed_args(args: list[KApply]) -> KApply:
         res = KApply('.List{"_,__EVM-ABI_TypedArgs_TypedArg_TypedArgs"}_TypedArgs')
-        for i in reversed(args):
-            res = KApply('_,__EVM-ABI_TypedArgs_TypedArg_TypedArgs', [i, res])
+        for arg in reversed(args):
+            res = KEVM.to_typed_arg(arg, res)
         return res
 
+    @staticmethod
+    def to_typed_arg(arg: KApply, res: KApply) -> KApply:
+        if arg.label == 'abi_type_tuple':
+            # TODO add tuple TypedArgs
```

Comment:
These TODOs could get lost in the code base. Would you open an issue to track them?

Reply:
Right, I meant to actually generate the `abi_type_tuple` klabel, it's part of https://github.com/runtimeverification/evm-semantics/pull/2051/commits/acac6c9a26e1445a33adf6532c150f51a3d455bf
Thanks!